hardhat.config.js

```diff
@@ -1,6 +1,6 @@
 require("@nomiclabs/hardhat-waffle");
 require("dotenv").config({path: "./.env"});
-require("hardhat-deploy");
+// require("hardhat-deploy");
```

Comment:
any reason why we keep it commented out? pls delete it or uncomment

Reply:
I guess I should also delete the `// require("hardhat-gas-reporter");` line that is commented below as well?

Reply:
Done